include/deal.II/fe/fe_bernardi_raugel.h

```diff
@@ -131,6 +131,24 @@ class FE_BernardiRaugel : public FE_PolyTensor<dim>
    */
   void
   initialize_support_points();
+
+  /**
+   * Fill the values Bernardi-Raugel polynomials, but replace the normal
+   * vector from the PolynomialsBernardiRaugel class with the physical
+   * normal vector from @p cell.
```

Comment:
Can you explain why that is correct? I think what you want to say instead of the first sentence is that this function calls the one in the base class but then overwrites the normal vectors. So the comment should explain why the normals filled by the base class aren't correct.

Reply:
Got it. I added a little more detail in there.